API_Guidelines.md

```diff
@@ -72,49 +72,42 @@ Payload
 
 ## PUT
 
-> `PUT` requests update records that already exist with new information
+> `PUT` requests update records that already exist with new or updated information
 
-### Example
-
-#### Current model:
-```javascript
-{
-    "bands": {
-        "id": 8,
-        "name": "Radiohead",
-        "bestAlbum": "OK Computer",
-        "bestBandEver": false
-    }
-}
-```
+### Request
+URL
+:   `apiHost.com/movies/2`
 
-#### Update is desired:
+Ember Data Method
+:   
 ```javascript
-set(model, 'bestBandEver', true);
-model.save(); // This initializes the PUT request
+//lookup record in the local store
+let movie = this.store.findRecord('movie', 2); // returns record of {"id": 2, "title": "Goodfellas", "year": "1990"}
```

Comment:
As long as we are consistent throughout our documentation, I can see the argument for or against it. In the *POST*  section, I use `get(this, 'store')`, and in the ERRORS section, `Ember.get(this, 'model.errors.first-name')` is used.